### ISSUE TITLE
Embedder can't embed text files

### ISSUE THREAD

**Issue by vrobel** [johnlindquist/open-source-plugins#6]
As all works fine for PNG, JPG files it doesn't do anything while I select css or xml file.

**Comment by johnlindquist** [comment]
Yeah, I only add "binary" files for now. I'll look into support for other file types. I may have to make a form that let's you put in *.css, *.xml, etc so you can determine it for yourself.

**Comment by vrobel** [comment]
Thanks John
